.github/workflows/test.yml

```diff
@@ -39,9 +39,11 @@ jobs:
       - uses: actions/cache@v3
         with:
           path: '**/node_modules'
-          key: yarn-${{ matrix.os }}-${{ matrix.node }}-${{ hashFiles('yarn.lock') }}
-          restore-keys: yarn-${{ matrix.os }}-${{ matrix.node }}
-      - run: yarn install --network-timeout 1000000 --frozen-lockfile
+          key: pnpm-${{ matrix.os }}-${{ matrix.node }}-${{ hashFiles('pnpm-lock.yaml') }}
+          restore-keys: pnpm-${{ matrix.os }}-${{ matrix.node }}
+      - name: install pnpm
+        run: npm i -g pnpm
```

Comment:
Lets make sure to use `7.24.2` for all our installs so it matches what corepack would have used